### ISSUE TITLE
So it's not using basic packet signatures ?

### ISSUE THREAD

**Issue by ExtReMLapin** [sieveandsand/Hacking_X1_Pro#1]
If you can sniff the packet, it seems clear that there is little to no security here.
It means anyone "near" a e-sk8er could just inject commands and possibly hurt the skater ?

**Comment by sieveandsand** [comment]
You are right! There are no packet security in place. As long as you know what the address is, you can read the packets without any obstruction. It's a huge vulnerability and can definitely be exploited in harm's way.
